desktop/release-notes.md

```diff
@@ -24,13 +24,92 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.16.0
+
+{% include release-date.html date="2022-01-12" %}
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Extensions have moved from Beta to GA
+- Quick Search has moved from experimental to GA
+- Extensions are now included in Quick Search
+- Analyzing large images is now up to 4x faster
+
+### Upgrades
+
+- [Compose v2.15.0](https://github.com/docker/compose/releases/tag/v2.15.0)
+- [Containeld v1.6.14](https://github.com/containerd/containerd/releases/tag/v1.6.14)
+- [Docker Engine v20.10.22](https://docs.docker.com/engine/release-notes/#201022)
+- [Docker Scan v0.23.0](https://github.com/docker/scan-cli-plugin/releases/tag/v0.23.0)
+- [Go 1.19.4](https://github.com/golang/go/releases/tag/go1.19.4)
+
+### Bug fixes and enhancements
+
+#### For all platforms
```

Comment:
This PR fixes an important issue https://github.com/docker/pinata/pull/19991 on the Container overview page. 
There's no corresponding Github issue but it's been reported several times on feedback-channel Slack and also [by our client](https://docker.slack.com/archives/C0W4XJVFW/p1672985651503269). I'd put it here, something like
```
- Fixed "An error occured" with Container overview screen
```
WDYT @lizzthabet @landism ?

Reply:
thanks for calling this out, @trungutt! i might be more slightly more specific:

```
- Fixed the blank "An error occurred" container list on the Container Overview screen 
```